src/Compilers/Core/Portable/Operations/OperationWalker.cs

```diff
@@ -16,10 +16,15 @@ public abstract class OperationWalker : OperationVisitor
         {
             foreach (var operation in operations)
             {
-                Visit(operation);
+                VisitArrayElement(operation);
             }
         }
 
+        internal void VisitArrayElement<T>(T operation) where T : IOperation
```

Comment:
It is necessary as it is also passed as a delegate to VisitArrayCommon in the OperationTreeVerifier.